src/main/java/com/expedia/haystack/dropwizard/configuration/TracerFactory.java

```diff
@@ -53,6 +51,9 @@
     @NotEmpty
     private List<DispatcherFactory> dispatchers = ImmutableList.of(new RemoteDispatcherFactory());
 
+    @Valid
+    private IdGenerator idGenerator;
```

Comment:
Can we add LongIdGenerator as default?

```private IdGenerator idGenerator = new LongIdGenerator()```

Reply:
Hi @mchandramouli, I did consider giving this a default value but decided against it as the `Tracer` (that's being built by this factory) already has a sensible default if an `idGenerator` is not provided (or is null).  By adding another default here I was concerned that the two default values may get out of sync.

Happy to implement your suggestion, just thought i'd explain my rationale for not providing one.

Reply:
@amcghie good point. I am resolving this comment and merging your PR. when you get a chance can you issue a PR for README